feature/experimental/policy/otg_tests/prefix_set_test/prefix_set_test.go

```diff
@@ -62,9 +65,12 @@ func TestPrefixSet(t *testing.T) {
 
 	// replace the prefix-set by replacing an existing prefix with new prefix
 	v4PrefixSet = ds.GetOrCreatePrefixSet(prefixSetA)
-	v4PrefixSet.SetMode(oc.PrefixSet_Mode_IPV4)
+	if !deviations.SkipPrefixSetMode(dut) {
+		v4PrefixSet.SetMode(oc.PrefixSet_Mode_IPV4)
+	}
 	v4PrefixSet.GetOrCreatePrefix(pfx1, mskLen)
 	v4PrefixSet.GetOrCreatePrefix(pfx3, mskLen)
+	v4PrefixSet.DeletePrefix(pfx2, mskLen)
```

Comment:
we are doing `gnmi.Replace`, and our expectation is it will replace old prefixes with new ones so we don't want to explicitly remove the `pfx2` here

Reply:
`DeletePrefix` is not translated into the gnmi.Delete call; this is about the data structure (`ds`) you're working with which is not re-initialized between the assignments. `ds.GetOrCreatePrefixSet(prefixSetA)` returns the structure that was configured in the previous step.